### PR TITLE
Fix namespaces

### PR DESCRIPTION
**Pull request by abhipsaMisra**
These public types don't require their own namespaces

### CODE REVIEW

File: iothub/device/src/Transport/Mqtt/IWillMessage.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.Azure.Devices.Client.Transport.Mqtt
+namespace Microsoft.Azure.Devices.Client
```
Comment (abhipsaMisra) [inline iothub/device/src/Transport/Mqtt/IWillMessage.cs:4]: The amqp public types are under the "root" namespace, so moving this there as well

File: iothub/device/src/FileUpload/FileUploadCompletionNotification.cs [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 using Newtonsoft.Json;
 
-namespace Microsoft.Azure.Devices.Client.Transport
+namespace Microsoft.Azure.Devices.Client
```
Comment (abhipsaMisra) [inline iothub/device/src/FileUpload/FileUploadCompletionNotification.cs:6]: This, and below - they aren't transport specific types really.

**Review by drwill-ms — APPROVED** [review]

**Review by tmahmood-microsoft — APPROVED** [review]

**Review by andyk-ms — APPROVED** [review]

**Review by patilsnr — APPROVED** [review]

**Pull request merged**